src/client.js

```diff
@@ -461,10 +461,25 @@ function SugarApi(args) {
                 };
 
                 if (!('crosstab' in window) || !crosstab.supported) {
+
+                    // prevents concurrent events from multiple tabs asking for a
+                    // refresh token
+                    if (_refreshingToken) {
+                        return;
```

Comment:
Can you please describe how it works?
Value of a variable is not sharing between browser tabs, so `_refreshingToken` is always `undefined` here, isn't it?

Reply:
I think so. We don't use `crosstab` functionality correctly here. We send refresh token simultaneously for different tabs in this place. For regular behaviour, we consider refresh token process could already be run and doesn't allow sending request from different tab. here: https://github.com/sugarcrm/ventana/blob/master/src/client.js#L2146
But in case _handleErrorAndRefreshToken method is called we don't provide the same logic